scripts/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/plot_presenter.py

```diff
@@ -43,3 +46,18 @@ def clear_plot(self):
             self.model.remove_all_workspaces_from_plot(ax)
         self.view.clear_figure()
         self.view.update_fitbrowser()
+
+    def do_sequential_fit(self, ws_list):
```

Comment:
Thanks I'd come across the curiously named `PlotPeaksByLogValue`, but it does not use the proton charge weighted average of the log value  (I suppose I could have changed this though), and as you say I wanted to keep the browser updated and also populate the internal dicts etc. with something in the same format as a normal fit. It is actually quite easy for me to report the fit status and also return chi squared etc in the model table (I considered doing this but didn't want to overload the table with info)? The chi-sq value is of course accessible in the individual output workspaces produced by the call to `Fit`  (unfortunately I suspect many users operate on a 'chi-by-eye' principle...)

Reply:
I have added the cost function to the model table and also print each workspace name prior to the fit so that the user can inspect the fit status (the fit algorithm prints failures appear at warning level of log)